ql/src/java/org/apache/hadoop/hive/ql/exec/TaskRunner.java

```diff
@@ -74,6 +75,7 @@ public boolean isRunning() {
 
   @Override
   public void run() {
+    LogUtils.registerLoggingContext(ss.getConf());
```

Comment:
Should use tsk.getConf() otherwise the configs provided via WITH clause won't be there.

Reply:
done